notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -279,27 +385,31 @@
   {
    "attachments": {},
    "cell_type": "markdown",
+   "id": "22d445d7",
    "metadata": {},
    "source": [
     "![Counterfactual of datapoint 43 selected on the counterfactuals scatter plot with axes \"Predicted Y\" and \"Index\". Predicted Y is 115.4](./img/regression-decision-making-4.png)"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "b4f78fd8",
    "metadata": {},
    "source": [
     "Note that this result does not mean that reducing `bmi` and `s5` *causes* the diabetes progression score to go down. It simply decreases the model prediction. To investigate causal relationships, continue reading:"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "b134cdb5",
```

Comment:
is this a new thing? I've never seen these IDs before. Do they go away if you do the clear output command?

Reply:
Not sure why these got update. I am not sure how to correct these.